Makefile.toml

```diff
@@ -148,7 +155,7 @@ run_task = "ci-flow"
 description = "Install/build dependencies for api tests"
 workspace = false
 private = true
-script = [ "( cd ./api_tests && yarn install; yarn run gen; )" ]
+script = [ "( cd ./api_tests && yarn install;)" ]
```

Comment:
❓ The `gen` script was renamed to `postinstall` - but as far as I could see `postinall` is never triggered. Is it triggered implicitly somehow, am I missing something?

Reply:
It is indeed triggered implicitly after an install